src/tests/SharpRaven.UnitTests/Data/HttpRequestBodyConverterTests.cs

```diff
@@ -60,6 +63,52 @@ public void Convert_Form_ReturnsForm()
 
 
         [Test]
+        [TestCaseSource(typeof(JsonTestCase), "TestCasesContentType")]
```

Comment:
Wouldn't it be easier to read if you just used `[TestCase()]` with the content type strings directly in the attribute, instead of extracting them to a source type? I'd understand it if the source data was more complex, but since it's just a string... :smiley: